scripts/report_coverage.sh

```diff
@@ -0,0 +1,56 @@
+mkdir -p coverage/merged
+go tool covdata merge -i=coverage/e2e,coverage/ut -o coverage/merged
+
+go tool covdata textfmt -i coverage/merged/ -o profile.txt
+
+cat profile.txt\
+	| grep -v github.com/ava-labs/avalanche-cli/internal/mocks > profile.tmp
```

Comment:
lets filter out just mocks, not github.com/ava-labs/avalanche-cli/internal/mocks, a new mock was added in a different location